### PR TITLE
Add content-aside for article page

### PR DESCRIPTION
**Pull request by Cosmin-Daniel**

### CODE REVIEW

**Review by smoqadam — APPROVED** [review]

File: src/ViewModel/Button.php [hunk 1]
```diff
@@ -150,6 +151,11 @@ public static function action(
                 ]
             );
             $button->classes .= ' icon icon-' . $variant;
+
+            if ($variant == self::ACTION_VARIANT_COMMENT) {
+                $button->isHypothesisTrigger = true;
+                $button->text = "Comment<span aria-hidden='true'><span data-visible-annotation-count></span> </span><span class='visuallyhidden'>Open annotations (there are currently <span data-hypothesis-annotation-count>0</span> annotations on this page). </span>";
```
Comment (nlisgo) [inline src/ViewModel/Button.php:157]: text can be removed. Journal can supply all of that text.

We need to adjust ButtonTest.php

File: src/ViewModel/ContentHeaderNew.php [hunk 2]
```diff
@@ -49,7 +50,8 @@ public function __construct(
         MetaNew $meta = null,
         Doi $doi = null,
         string $licence = null,
-        AudioPlayer $audioPlayer = null
+        AudioPlayer $audioPlayer = null,
+        bool $hasAside = false
```
Comment (nlisgo) [inline src/ViewModel/ContentHeaderNew.php:54]: We need to amend the ContentHeaderNewTest

**Review by nlisgo — CHANGES_REQUESTED** [review]

File: src/ViewModel/Button.php (hunk 1 above)
Comment (Cosmin-Daniel) [reply src/ViewModel/Button.php]: text has been removed and tests have been updated.

File: src/ViewModel/ContentHeaderNew.php (hunk 2 above)
Comment (Cosmin-Daniel) [reply src/ViewModel/ContentHeaderNew.php:54]: Done. The tests cover now hasAside.

File: tests/src/ViewModel/ContentAsideTest.php [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                new ContentAside(
+                    new ContentAsideStatus(
+                        'Research article',
+                        'The author(s) have declared this to be the current/final version.',
+                        new Link('About eLife\'s process', '#')
+                    ),
+                    new ButtonCollection([Button::action('text', '#')], true),
+                    ContextualData::withMetrics(['foo']),
+                    DefinitionList::timeline(['foo' => ['bar']]),
+                    ListingTeasers::basic([
```
Comment (nlisgo) [inline tests/src/ViewModel/ContentAsideTest.php:135]: Switch to seeMoreLink

File: src/ViewModel/Button.php [hunk 4]
```diff
@@ -150,6 +151,10 @@ public static function action(
                 ]
             );
             $button->classes .= ' icon icon-' . $variant;
+
+            if ($variant == self::ACTION_VARIANT_COMMENT) {
```
Comment (nlisgo) [inline src/ViewModel/Button.php:155]: strict equals please: `===`

**Review by nlisgo — APPROVED** [review]
Approved with one minor change needed. See comment.

File: src/ViewModel/Button.php (hunk 4 above)
Comment (Cosmin-Daniel) [reply src/ViewModel/Button.php]: Done.

**Pull request merged**